### PR TITLE
Contact api

### PR DESCRIPTION
**Pull request by kiruthikaSukumar**

### CODE REVIEW

File: controllers/api/enqiryController.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    const existingEnquiry = await EnquiryModel.findOne({
+      $or: [{ phoneNumber }, { email: requestData.email.trim() }],
+    });
+
+    if (existingEnquiry) {
+      return res.json({
+        message: "Phone number or email already exists.",
+        success: true,
+        statusCode: 200,
```
Comment (praveensumathi) [inline controllers/api/enqiryController.js:28]: change the error code

File: controllers/api/enqiryController.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+/**
+ * @param {Request} req - The Express request object
+ * @param {Response} res - The Express response object
+ */
+
+exports.createEnquiry = async (req, res, next) => {
+  try {
+    const requestData = req.body;
+    const phoneNumber =
+      typeof requestData.phoneNumber === "string"
```
Comment (praveensumathi) [inline controllers/api/enqiryController.js:16]: remove this

File: controllers/api/enqiryController.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        message: "Phone number or email already exists.",
+        success: true,
+        statusCode: 200,
+      });
+    }
+
+    const enquiry = {
+      childName: requestData.childName.trim(),
+      parentName: requestData.parentName.trim(),
+      phoneNumber,
```
Comment (praveensumathi) [inline controllers/api/enqiryController.js:35]: change to requestData.phonenumber

File: routes/enquiry.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+var express = require("express");
+var router = express.Router();
+
+const apienquiryController = require("../controllers/api/enqirycontroller");
```
Comment (praveensumathi) [inline routes/enquiry.js:4]: use CamelCase, and enquiryControllr camel case check

File: controllers/api/enqiryController.js (hunk 1 above)
Comment (kiruthikaSukumar) [reply controllers/api/enqiryController.js:28]: changed

**Review by praveensumathi — APPROVED** [review]

**Pull request merged**